open-craft/opencraft
# [BB-4230] feature: Support usage of `EDXAPP_FEATURES_EXTRA`.

The user is now allowed to set feature flags through OCIM as he would when using the `configuration` playbooks.

Associated tickets:
    * [BB-4230](https://tasks.opencraft.com/browse/BB-4230)

h1. Testing

1. Launch vanilla instance from this branch
2. Launch vanilla instance from [current `master` branch](https://github.com/open-craft/opencraft/tree/f4d442b97ee32fdadda8325b85aa873365fea225)
3. Verify that feature configurations are identical
